src/raftstore/store/config.rs

```diff
@@ -167,6 +169,7 @@ impl Default for Config {
             max_leader_missing_duration: ReadableDuration::hours(2),
             abnormal_leader_missing_duration: ReadableDuration::minutes(10),
             peer_stale_state_check_interval: ReadableDuration::minutes(5),
+            leader_transfer_max_log_lag: 10,
```

Comment:
Let's set it to a larger value.

Reply:
How large should it be? Also, are there better solutions like based on time instead of an absolute value?

Reply:
How to measure time of future? Maybe a force flag can be used to force it to transfer leader by ignoring log gap. So PD can set the flag when it thinks it's necessary.